Python/Product/PythonTools/PythonTools/Commands/CurrentEnvironmentCommand.cs

```diff
@@ -32,29 +32,28 @@
 namespace Microsoft.PythonTools.Commands {
     class CurrentEnvironmentCommand : OleMenuCommand {
         private readonly IServiceProvider _serviceProvider;
-        private readonly EnvironmentSwitcherManager _envSwitchMgr;
-        private readonly IPythonToolsLogger _logger;
+        private EnvironmentSwitcherManager _envSwitchMgr = null;
```

Comment:
Refactored this a bit as the GetPythonToolsService was taking a long time in package load.